src/AspNet.Security.OAuth.LinkedIn/LinkedInAuthenticationHandler.cs

```diff
@@ -24,7 +25,15 @@ public LinkedInAuthenticationHandler([NotNull] HttpClient client)
 
         protected override async Task<AuthenticationTicket> CreateTicketAsync([NotNull] ClaimsIdentity identity,
             [NotNull] AuthenticationProperties properties, [NotNull] OAuthTokenResponse tokens) {
-            var request = new HttpRequestMessage(HttpMethod.Get, Options.UserInformationEndpoint);
+            if (Options.Fields.Count == 0) {
```

Comment:
@jerriep do you happen to know if specifying at least one explicit field is mandatory? I'd expect LI to simply return basic info in this case.

@ericgreenmix if adding at least one field is mandatory, I'd recommend moving this check to `LinkedInAuthenticationMiddleware`'s constructor to crash at startup time instead of during callback requests.